File: app/src/main/java/com/cogu/spylook/model/relationships/ContactoAnotacion.java
package com.cogu.spylook.model.relationships;

import androidx.room.Embedded;
import androidx.room.Relation;

import com.cogu.spylook.model.entity.Anotacion;
import com.cogu.spylook.model.entity.Contacto;

import java.util.List;

public class ContactoAnotacion {
    @Embedded
    public Contacto contacto;

    @Relation(parentColumn = "id", entityColumn = "idContacto")
    public List<Anotacion> anotaciones;
}

File: app/src/main/java/com/cogu/spylook/model/relationships/ContactoAnotacion.kt
package com.cogu.spylook.model.relationships

import androidx.room.Embedded
import androidx.room.Relation
import com.cogu.spylook.model.entity.Anotacion
import com.cogu.spylook.model.entity.Contacto

class ContactoAnotacion {
    @JvmField
    @Embedded
    var contacto: Contacto? = null

    @JvmField
    @Relation(parentColumn = "id", entityColumn = "idContacto")
    var anotaciones: MutableList<Anotacion?>? = null
}
